PublishParasoftTestResults/publishtestresults.ts

```diff
@@ -258,6 +258,7 @@ function mapToAnalyzer(node: any) {
 }
 
 function processResults(inputReportFiles: string[], index: number){
+    addDocURIInSarif(inputReportFiles[index]);
```

Comment:
I think this process should only happen when we try to transform a xml static analysis report to SARIF report.
Consider only do it inside that condition and see if we can modify the JSON before we call "fs.writeFileSync(outPath, result.principalResult);" inside "transform" method.